aptos-move/e2e-move-tests/src/lib.rs

```diff
@@ -60,5 +60,10 @@ pub(crate) fn build_package_with_compiler_version(
 ) -> anyhow::Result<BuiltPackage> {
     let mut options = options;
     options.compiler_version = Some(compiler_version);
+    if options.compiler_version.unwrap() != CompilerVersion::V1 {
+        options.language_version = Some(move_model::metadata::LanguageVersion::latest_stable());
+    } else {
+        options.language_version = Some(move_model::metadata::LanguageVersion::V1);
+    }
```

Comment:
nit: we may want to add something like `infer_language_version(compiler_version: CompilerVersion)` if this pattern is going to show up regularly (which it might, similarly to inferring bytecode version).